Source/WebGPU/WebGPU/WebGPU.h

```diff
@@ -216,29 +216,32 @@ typedef enum WGPUDeviceLostReason {
 typedef enum WGPUErrorFilter {
```

Comment:
I assume this file is literally unchanged from upstream?

Reply:
Yes that is correct